test/useLocalStore.test.tsx

```diff
@@ -236,4 +238,210 @@ describe("is used to keep observable within component body", () => {
         fireEvent.click(div)
         expect(div.textContent).toBe("initial - 10later - 20")
     })
+
+    describe("with props", () => {
+        it("and useObserver", () => {
+            let counterRender = 0
+            let observerRender = 0
+
+            function Counter({ multiplier }: { multiplier: number }) {
+                counterRender++
+
+                const store = useLocalStore(props => ({
```

Comment:
When updating the Readme, we should make it abundantly clear that `props` isn't supposed to be destructured

Reply:
Before updating the readme, it is probably the best to wait until #140 is merged to avoid potential merge issues